AnkiDroid/src/main/java/com/ichi2/anki/Whiteboard.kt

```diff
@@ -223,6 +223,12 @@ class Whiteboard(activity: AnkiActivity, handleMultiTouch: Boolean, inverted: Bo
      */
     override fun onSizeChanged(w: Int, h: Int, oldw: Int, oldh: Int) {
         super.onSizeChanged(w, h, oldw, oldh)
+        Timber.w("Whiteboard::onSizeChanged width:$w height:$h")
```

Comment:
```suggestion
        Timber.i("Whiteboard::onSizeChanged width:$w height:$h")
```
Here it's just an info, so use `Timber.i`. Below it's a potential error situation where we could recover, so it's a warn `Timber.w`